mypy/test/testconfigparser.py

```diff
@@ -0,0 +1,168 @@
+import os
+import tempfile
+from unittest import TestCase, main
+
+from mypy.config_parser import parse_config_file
+from mypy.options import Options
+
+
+class TestConfigParser(TestCase):
+    def test_parse_config_file_with_single_file(self) -> None:
+        """A single file should be correctly parsed."""
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            config_path = os.path.join(tmpdirname, "test_config.ini")
```

Comment:
Instead of using old string-based APIs, use `pathlib`. Although, in the context of pytest, `tmp_path` is what you're looking for.